src/DiskIO/DiskDaemon/DiskdAction.cc

```diff
@@ -65,13 +65,13 @@ DiskdAction::Create(const Mgr::CommandPointer &aCmd)
 DiskdAction::DiskdAction(const Mgr::CommandPointer &aCmd):
     Action(aCmd), data()
 {
-    debugs(79, 5, HERE);
+    debugs(79, 5, "");
```

Comment:
There are two kinds of branches in this context:

1. Branches of the first kind simply track master. These branches do not care whether the changes were done manually or automatically. They simply merge/rebase as needed.
2. Branches of the second kind do not follow master (or sync with it very infrequently) but still take many isolated changes from master. These branches can run the `HERE` removal script without significant problems and then continue to cherry pick future master changes as needed. On the other hand, these branches cannot apply widespread manual changes without wasting a lot of time on resolving a lot of conflicts.

> By combining the HERE change with the other polishing into one commit the branches then only face a conflict once IF they change that line themselves. Doing them separately will result in N conflicts for each affected line.

The difference you are describing is irrelevant because this PR does not suggest or imply doing "other polishing" or N "separate" manual changes. This PR does _no_ manual HERE-line changes at all, and I suggest that we continue to avoid manual polishing changes after this PR (unless the lines need to be changed for other reasons).